src/main/java/pl/put/poznan/json/tools/model/decorations/AttributesPicker.java

```diff
@@ -13,10 +17,29 @@ public AttributesPicker(JsonObject jsonObject, String[] attributes) {
         this.attributes = attributes;
     }
 
-    public String getJson() throws WrongInputException {
-        //TODO
-        //tu zmienic jsonObject.getJson() na taki tylko z wybranymi atrybutami
-        return jsonObject.getJson();
+    public String getJson() throws WrongInputException, ParseException {
+        return pickAttributes(jsonObject.getJson());
     }
 
+    private String pickAttributes(String json) throws ParseException {
```

Comment:
Sugeruję zmienić nazwę metody na getJsonWithAttributes() a metodę prywatną poniżej nazwać pickAttributes(). To lepiej obrazuje co robią metody